Facebook-AutoLogin/README.md

```diff
@@ -0,0 +1,34 @@
+# Facebook-AutoLogin
+</br>
```

Comment:
Not needed

Reply:
I meant the <br> tag.

Reply:
@bismitaguha Made all the changes , can you re-review the PR